index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+    <style>
+        
+    </style>
+</head>
+<body>
+    <div class="container">
+       <div class="header-bg">
+           <img class="Ellipse12" src="images/Ellipse 12.png" alt="">
+            <img class="Ellipse13" src="images/Ellipse 13.png" alt="">
+            <img class="Ellipse4" src="images/Ellipse 4.png" alt="">
+            <img class="Rectangle5" src="images/Rectangle 5.png" alt="">
+            <img class="Ellipse3" src="images/Ellipse 3.png" alt="">
+            <img class="Rectangle4" src="images/Rectangle 4.png" alt="">
+            <img class="Rectangle7" src="images/Rectangle 7.png" alt="">
+            <img class="Ellipse5" src="images/Ellipse 5.png" alt="">
+            <img class="Ellipse11" src="images/Ellipse 11.png" alt="">
+            <img class="Rectangle6" src="images/Rectangle 6.png" alt="">
+            <img class="Ellipse15pro" src="images/Ellipse15pro.png" alt="">
+        </div>
+        <header class="navbar">
+            <a class="logo" href="#"><img src="images/My Logo.png" alt=""></a>
+            <nav class="menu-bar">
+                <a href=""><img src="images/Union.png" alt=""></a>
+            </nav>
+        </header>
+
+        <section class="main">
+            <div class="main-header">
+                <h2>I'm Alex Waweru.
+                    Glad to see you!</h2>
+            </div>
+
+            <div class="main-content">
+                <p>
+                    I'm a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, 
+                     don't hesitate to contact me.
+                </p>
+            </div>
+
+            <div class="connect">
+                <a href="#">LET'S CONNECT</a>
+            </div>
+            <div class="icons">
+                <a href="#" class="twitter" ><img src="images/twitter.svg" alt=""></a>
+                <a href="#" class="vector1"  ><img src="images/Vector.png" alt=""></a>
+                <a href="#" class="vector2"  ><img src="images/Vector (1).png" alt=""></a>
+                <a href="#" class="vectorM" ><img src="images/VectorM.png" alt=""></a>
+                <a href="#" class="github" ><img src="images/github.svg" alt=""></a>
+                <a href="#" class="vectorsixer" ><img src="images/Vectorsixer.png" alt=""></a>
+            </div>
+
+        </section>
+
+
+
+        
+
+  
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you remove all the empty code lines from your code base to make it clean and professional